### PR TITLE
Add sabayon-brokenlinks script

### PR DESCRIPTION
**Pull request by geaaru**

### CODE REVIEW

File: sabayon-brokenlinks [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+# Authors: Sabayon Team 2019
+
+set -e
+
+# Accepted values: x86_64, armv7l
+export SAB_ARCH="${SAB_ARCH:-}"
+export DRY_RUN=${DRY_RUN:-0}
+export DEBUG="${DEBUG:-}"
+export FORCE_MANUAL="${FORCE_MANUAL:-0}"
+export GENTOO_TOOL="${GENTOO_TOOL:-0}"
```
Comment (mudler) [inline sabayon-brokenlinks]: Wouldn't be `USE_EQUO` a better choice as it is more consistent here as we use it in other parts?

File: sabayon-brokenlinks [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+    echo "Sabayon Broken Links analyzer.
+Sabayon Team (2019)
+
+$0 [opts]
+
+Available options:
+
+-h|--help         This message.
+--dry-run         Dry run execution.
+--force-manual    Force remove of links to remove manually.
```
Comment (mudler) [inline sabayon-brokenlinks]: Can't tell what is supposed to do without looking at the code, is that a ```--pretend``` candidate? :smiley:

File: sabayon-brokenlinks (hunk 1 above)
Comment (geaaru) [reply sabayon-brokenlinks]: Ok. I copy that.

File: sabayon-brokenlinks (hunk 2 above)
Comment (geaaru) [reply sabayon-brokenlinks]: `--dry-run` permits to analyze directories without removing nothing. Only prints broken links.
`--force-manual`: there are two type of broken links:

a) links where it's possible to identify a package. The default logic for this are:

   - if the link is related to an existing package with the found version, just said: try to reinstall package xx
  - if the link is related to a package installed with a more upgraded version, just by default remove that link at the end of the analysis (without asking user)

b) links broken without the possibility to identify the source package that has to generate the link (in a similar way of `equo s orphans`). In this case, the default logic is: not remove the link but print at the end of the analysis the command to use for removing the file. With option `--force-manual` also this files/links are removed.

Comment (mudler) [reply sabayon-brokenlinks]: What I mean is, I would stick with a generic `--force`, as I don't see any other `--force` already being used in another arg.

Comment (geaaru) [reply sabayon-brokenlinks]: Done ;)

Comment (mudler) [reply sabayon-brokenlinks]: Awesome! :)

Comment (mudler) [reply sabayon-brokenlinks]: Thanks!

**Pull request merged**